timeComplexity.js

```diff
@@ -0,0 +1,11 @@
+function containsDuplicates(arr) {
+    for (let i = 0; i < arr.length; i++) {
+      for (let j = 0; j < arr.length; j++) {
+        if (i !== j && arr[i] === arr[j]) {
+          return true;
+        }
+      }
+    }
+    return false;
+  }
```

Comment:
I wonder if suggestions of this format are better to use 🤔 

```suggestion
function containsDuplicates(arr) {
  const set = new Set();
  for (let i = 0; i < arr.length; i++) {
    if (set.has(arr[i])) {
      return true;
    }
    set.add(arr[i]);
  }
  return false;
  }
```